docs/README.md

```diff
@@ -1,11 +1,36 @@
 # No Chat Reports
 
-**Current Version:** 1.19.3-v1.19.0
+A mod that strips cryptographic signatures from chat messages and disables Player Chat Reporting in Minecraft.   
+GitLab Mirror: https://gitlab.com/Aizistral-Studios/No-Chat-Reports
 
-**Minecraft Version:** 1.19.3
+## Installation and Use
 
-Strips cryptographic signatures from player messages, making it impossible to track and associate them with your Mojang/Microsoft account, as well as use Player Chat Reporting feature.
+This mod supports both Forge and Fabric and can be installed on the client, server, or both. It will function differently depending on where it is installed:
 
-**GitLab Mirror:** https://gitlab.com/Aizistral-Studios/No-Chat-Reports
+1. **Only on the client**: The client will refuse to send the account's public key to the server, and signatures will be stripped from the messages that are sent. This way it won't be possible to report your messages, as there will be no proof they were actually sent from your account. The server will relay them unless the "enforce-secure-profile" option is set to true (which it is by default since 1.19.1), in which case you will not be able to join unless you agree to send signed messages (the mod will display a warning screen).
```

Comment:
```suggestion
1. **Only on the client**: The client will refuse to send the account's public key to the server, and signatures will be stripped from the messages that are sent. This way it won't be possible to report your messages, as there will be no proof they were actually sent from your account. The server will relay them unless the `enforce-secure-profile` option is set to true (which it is by default since 1.19.1), in which case you will not be able to send messages unless you agree to sign them (the mod will display a warning screen).
```

Accurate as of 1.19.3